meson.build

```diff
@@ -0,0 +1,123 @@
+project('libtexpdf', 'c', version : '2.4.6', default_options : ['c_std=gnu11'])
+
+pkg = import('pkgconfig')
+
+compiler = meson.get_compiler('c')
+
+conf = configuration_data()
+conf.set('version', meson.project_version())
+
+check_headers = ['dlfcn.h', 'inttypes.h', 'stdint.h', 'stdlib.h', 'string.h', 'sys/stat.h', 'sys/types.h', 'sys/wait.h', 'unistd.h']
+foreach header_name : check_headers
+    if compiler.has_header(header_name)
+        conf.set('HAVE_' + header_name.underscorify().to_upper(), 1)
+    endif
+endforeach
+
+if compiler.has_function('timezone', prefix : '#include <time.h>')
+    conf.set('HAVE_TIMEZONE', 1)
+endif
+
+if compiler.has_function('getenv', prefix : '#include <stdlib.h>')
+    conf.set('HAVE_GETENV', 1)
+endif
+
+if compiler.has_function('mkstemp', prefix : '#include <stdlib.h>')
+    conf.set('HAVE_MKSTEMP', 1)
+endif
+
+if compiler.has_member('struct tm', 'tm_gmtoff', prefix : '#include <time.h>')
+    conf.set('HAVE_TM_GMTOFF', 1)
+endif
+
+configure_file(input: 'config.h.meson', output: 'config.h', configuration: conf)
+
+libm = compiler.find_library('m', required : false)
+libdl = compiler.find_library('dl', required : false)
```

Comment:
AFAICT libdl isn't used anywhere.

Reply:
> AFAICT libdl isn't used anywhere.

Old good libdl I not even know what it does, just an obscure legacy of Linux.

Thank you for pointing out this, I removed that library from the meson config.